src/ParseServer.js

```diff
@@ -171,7 +177,11 @@ class ParseServer {
     const userController = new UserController(emailControllerAdapter, appId, { verifyUserEmails });
     const liveQueryController = new LiveQueryController(liveQuery);
 
+    if (settingsCacheOptions) {
+      cache.apps = PersistentSettingsStore(settingsCacheOptions, definedOptions);
```

Comment:
If settingsCacheOptions is false there is no persistence.  Otherwise, cache.apps.set() will trigger a pull from the database, update persisted settings with whatever is returned, then push to the database.  However, locked & defined properties won't be updated even if they exist in the database because I attach a dummy setter to those properties.